app/src/main/kotlin/com/simplemobiletools/smsmessenger/activities/MainActivity.kt

```diff
@@ -111,8 +111,8 @@ class MainActivity : SimpleActivity() {
         search_holder.setBackgroundColor(getProperBackgroundColor())
 
         val properPrimaryColor = getProperPrimaryColor()
-        no_conversations_placeholder_2.setTextColor(properPrimaryColor)
-        no_conversations_placeholder_2.underlineText()
+        start_conversation_placeholder.setTextColor(properPrimaryColor)
+        start_conversation_placeholder.underlineText()
```

Comment:
lets keep the old name for consistency, thats how it is called in most apps. And it makes sense here too.

Reply:
done :+1: